<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="main">
    <widget class="QPushButton" name="btnFuegoNuevo">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>340</y>
       <width>171</width>
       <height>131</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>9</y>
       <width>781</width>
       <height>461</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="Datos">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="lblTime">
        <property name="font">
         <font>
          <pointsize>40</pointsize>
          <stylestrategy>PreferDefault</stylestrategy>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="text">
         <string>Time</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::NoTextInteraction</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblDate">
        <property name="font">
         <font>
          <pointsize>40</pointsize>
          <stylestrategy>PreferDefault</stylestrategy>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="text">
         <string>Date</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::NoTextInteraction</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblPosition">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>40</pointsize>
          <stylestrategy>PreferDefault</stylestrategy>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="text">
         <string>Position</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::NoTextInteraction</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblStatus">
        <property name="font">
         <font>
          <pointsize>40</pointsize>
          <stylestrategy>PreferDefault</stylestrategy>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="text">
         <string>Status</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::NoTextInteraction</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblSVTS">
        <property name="font">
         <font>
          <pointsize>40</pointsize>
          <stylestrategy>PreferDefault</stylestrategy>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="text">
         <string>sat view/tack sawtooth</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::NoTextInteraction</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblLCD">
        <property name="font">
         <font>
          <pointsize>40</pointsize>
          <stylestrategy>PreferDefault</stylestrategy>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="text">
         <string>Diff (ns)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::NoTextInteraction</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblCLCD">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <stylestrategy>PreferDefault</stylestrategy>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="text">
         <string>Corrected Diff (ns)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::NoTextInteraction</set>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>lblDate</zorder>
     <zorder>lblTime</zorder>
     <zorder>lblPosition</zorder>
     <zorder>lblLCD</zorder>
     <zorder>lblCLCD</zorder>
     <zorder>lblSVTS</zorder>
     <zorder>lblStatus</zorder>
    </widget>
    <zorder>layoutWidget</zorder>
    <zorder>btnFuegoNuevo</zorder>
   </widget>
   <widget class="QWidget" name="satellites">
    <widget class="QPushButton" name="btnFuegoNuevo_2">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>340</y>
       <width>171</width>
       <height>131</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="SkyplotWidget" name="skyplot" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>400</width>
       <height>400</height>
      </rect>
     </property>
    </widget>
    <widget class="QRoundProgressBar" name="ch01" native="true">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>20</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
    </widget>
    <widget class="QRoundProgressBar" name="ch02" native="true">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>20</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
    </widget>
    <widget class="QRoundProgressBar" name="ch03" native="true">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>20</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
    </widget>
    <widget class="QRoundProgressBar" name="ch04" native="true">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>20</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
    </widget>
    <widget class="QRoundProgressBar" name="ch05" native="true">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>110</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
    </widget>
    <widget class="QRoundProgressBar" name="ch08" native="true">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>110</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
    </widget>
    <widget class="QRoundProgressBar" name="ch06" native="true">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>110</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
    </widget>
    <widget class="QRoundProgressBar" name="ch07" native="true">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>110</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
    </widget>
    <widget class="QRoundProgressBar" name="ch09" native="true">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>200</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
    </widget>
    <widget class="QRoundProgressBar" name="ch10" native="true">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>200</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
    </widget>
    <widget class="QRoundProgressBar" name="ch11" native="true">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>200</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
    </widget>
    <widget class="QRoundProgressBar" name="ch12" native="true">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>200</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="config">
    <widget class="QPushButton" name="btnFuegoNuevo_3">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>340</y>
       <width>171</width>
       <height>131</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="txtDebug">
     <property name="geometry">
      <rect>
       <x>223</x>
       <y>30</y>
       <width>561</width>
       <height>271</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="tabletTracking">
      <bool>true</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>191</width>
       <height>451</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pbSavePosition">
        <property name="text">
         <string>Save Position</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbSendPosition">
        <property name="text">
         <string>Send Position</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbSetDefaults">
        <property name="text">
         <string>Set Default</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbSurvey">
        <property name="text">
         <string>Survey</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_6">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_8">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_9">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_7">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QRoundProgressBar</class>
   <extends>QWidget</extends>
   <header>qroundprogressbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SkyplotWidget</class>
   <extends>QWidget</extends>
   <header>skyplotwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
